mesh_segmenter/CMakeLists.txt

```diff
@@ -4,10 +4,15 @@ project(mesh_segmenter)
 ## Compile as C++14,
 add_compile_options(-std=c++14)
 
-find_package(VTK 7.1 REQUIRED NO_MODULE)
+find_package(
+  VTK 7.1 REQUIRED NO_MODULE
+)
 include(${VTK_USE_FILE})
 
-find_package(catkin REQUIRED)
+find_package(catkin REQUIRED COMPONENTS
+    roscpp)
```

Comment:
I have not done a real review, but I wanted to comment that we intentionally do not have a ROS dependency in this package. You should make this a ros independent library and then make a node that wraps it in noether similar to what we have done with toolpath planning and segmentation

Reply:
I agree with @mpowelson  on this one